### PR TITLE
Use our own timestamp and duration message

### PR DESCRIPTION
**Pull request by vincent-psarga**
## Summary

Do not rely on Google's Timestamp message but user own format.
The duration will be expression with the same message (although named Duration for clarity)

## Types of changes

Not rely applying. Not a bug fix, neither a new feature. The duration field __might__ break previous version.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## Checklist:

- [ ] The change has been ported to Java.
- [ ] The change has been ported to Ruby.
- [ ] The change has been ported to JavaScript.
- [ ] The change has been ported to Go.
- [ ] The change has been ported to .NET.
- [ ] I've added tests for my code.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: cucumber-messages/messages.proto [hunk 1]
```diff
@@ -433,31 +458,32 @@ message TestStepMatched {
...
   TestResult testResult = 2;
-  google.protobuf.Timestamp timestamp = 3;
+  Timestamp timestamp = 3;
 }
 
 message TestResult {
   Status status = 1;
   string message = 2;
-  uint64 durationNanoseconds = 3;
+  uint64 durationNanoseconds = 3 [deprecated=true];
```
Comment (aslakhellesoy) [inline cucumber-messages/messages.proto]: Up until now we have not used protobuf's deprecated feature. Instead we are simply replacing the field with `reserved`. See the comment on line 7.

Once we have stuff in production we can consider `deprecated`, but for now I think it is best to remove it (more thorough cleanup, and no risks).

**Comment by aslakhellesoy** [comment]
I think we should include an explnanation about _why_ we are replacing the stdlib messages with our own.

File: cucumber-messages/messages.proto (hunk 1 above)
Comment (vincent-psarga) [reply cucumber-messages/messages.proto]: Works for me. I wasn't sure about Specflow handling of messages, but I think we can discuss it in the coming meeting with them.
I thought the deprecation would make it easier on their side :)

Comment (aslakhellesoy) [reply cucumber-messages/messages.proto]: I think deprecation will only make it easier to keep using the old field, which we want to avoid.

Comment (vincent-psarga) [reply cucumber-messages/messages.proto]: that's removed

File: cucumber-messages/go/go.mod [hunk 2]
```diff
@@ -3,6 +3,7 @@ module github.com/cucumber/cucumber-messages-go/v5
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
+	github.com/golang/protobuf v1.3.2 // indirect
```
Comment (aslakhellesoy) [inline cucumber-messages/go/go.mod]: Hmm I wonder where this comes from. We're using `gogo/protobuf`, not `golang/protobuf`...

**Comment by aslakhellesoy** [comment]
The reason the build is failing is that you have deleted `cucumber-messages/go/messages.pb.go` from git.

This file needs to be checked into git, so that consumers of the package can access the file. (Consumers will not run protobuf to generate it)

**Pull request merged**